### PR TITLE
[NUI] Support selector for PixelSize in TextLabelStyle and TextEditorStyle

### PR DESCRIPTION
**Pull request by rabbitfor**
Signed-off-by: Jiyun Yang <ji.yang@samsung.com>

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/Style/TextFieldStyle.cs [hunk 1]
```diff
@@ -438,15 +438,15 @@ public class TextFieldStyle : ViewStyle
         });
         /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public static readonly BindableProperty PixelSizeProperty = BindableProperty.Create(nameof(PixelSize), typeof(float?), typeof(TextFieldStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
+        public static readonly BindableProperty PixelSizeSelectorProperty = BindableProperty.Create("PixelSizeSelector", typeof(float?), typeof(TextFieldStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/BaseComponents/Style/TextFieldStyle.cs:441]: I think it would be better to use 'nameof(PixelSizeSelector)'

Comment (rabbitfor) [reply src/Tizen.NUI/src/public/BaseComponents/Style/TextFieldStyle.cs:441]: You are right. But unfortunately, there is no `PixelSizeSelector` in `TextField`.

Comment (huiyueun) [reply src/Tizen.NUI/src/public/BaseComponents/Style/TextFieldStyle.cs:441]: Done

**Review by huiyueun — APPROVED** [review]
Looks good to me

**Pull request merged**